renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
+  GLuint prevBinding;
+  m_Real.glGetIntegerv(eGL_ARRAY_BUFFER_BINDING, (GLint *)&prevBinding);
+
+  for(GLuint i = 0; i < ARRAY_COUNT(m_ClientMemoryVBOs); i++)
+  {
+    GLint enabled, size, stride;
```

Comment:
Maybe initialise these to 0 in case something goes wrong with the queries and the values aren't written.